string state_type
string dimension
int64 num_bfs
float64 dt

float64 tau
float64 alpha
float64 beta
float64 gamma

float64[] w

float64 init
float64 goal

